pkg/provider/azure_loadbalancer_repo_test.go

```diff
@@ -550,3 +550,200 @@ func TestIsNICPool(t *testing.T) {
 		})
 	}
 }
+
+func TestCheckRemoveOutdatedBasicLoadBalancers(t *testing.T) {
```

Comment:
this is great to have unit tests. Are we able to add an e2e test as well?

Reply:
It's hard to change the loadbalancersku in an e2e test, so I tested it manually.